src/shared/components/FormField/FormField.style.ts

```diff
@@ -0,0 +1,30 @@
+import { colors, sizes, typography } from '@/shared/theme'
+import styled from '@emotion/styled'
+import Text from '../Text'
+
+type FormFieldWrapperProps = {
+  fullWidth?: boolean
+}
+
+export const FormFieldWrapper = styled.div<FormFieldWrapperProps>`
+  margin-top: ${sizes(2)};
+  width: ${({ fullWidth }) => (fullWidth ? '100%' : '350px')};
+`
+
+export const FormFieldTitle = styled(Text)`
+  margin: 0;
+  line-height: ${sizes(5)};
+  word-wrap: break-word;
+`
+
+export const FormFieldDescription = styled(Text)`
+  margin-top: ${sizes(2)};
+  font-size: ${typography.sizes.subtitle2};
+  color: ${colors.gray[300]};
+  line-height: ${sizes(5)};
+  word-wrap: break-word;
```

Comment:
One thing I see in Figma in video upload flow is that the description in general doesn't span as much width as the input itself. Something around 80/90% I think? https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=3220%3A1730 Maybe we could add that?

Reply:
Had to change the input elements max-widths(and widths) as well. I decided to set the global variable for that. Not sure if it's a good idea - I can always revert that.